radicle-httpd/src/api/v1/projects.rs

```diff
@@ -275,24 +272,28 @@ async fn remote_handler(
 /// `GET /projects/:project/blob/:sha/*path`
 async fn blob_handler(
     Extension(ctx): Extension<Context>,
-    Path((project, sha, path)): Path<(Id, Oid, String)>,
+    Path((proj, sha, path)): Path<(Id, Oid, String)>,
 ) -> impl IntoResponse {
     let path = path.strip_prefix('/').ok_or(Error::NotFound)?;
     let storage = &ctx.profile.storage;
-    let repo = storage.repository(project)?;
-    let blob = radicle_surf::blob::blob(&repo.raw().into(), Some(Sha { sha }), path)?;
+    let repo = Repository::open(paths::repository(storage, &proj))?;
+    let blob = repo.blob(sha, &path)?;
+    let response = json!({
+        "path": &path,
+        "content": blob.content(),
+    });
```

Comment:
The actual JSON response might be different, depending on what `httpd` team wanted. This is just to show that I think `httpd` should define its JSON output format independent of `radicle-surf` API serialization. 

If there is some data needed by `httpd` JSON but missing from `radicle-surf` API, we can always fix that.

Reply:
Yeah I mostly agree with that, though I think that a good JSON format can work for more than one use-case, so it's a good aim to have.

Reply:
btw, is there a spec for `radicle-httpd` API? (like in swagger format or something similar?)

Reply:
No, but this is something we intended to create as things stabilize a little!